webapp/channels/src/components/with_tooltip/index.tsx

```diff
@@ -27,6 +29,8 @@ const WithTooltip = ({
     shortcut,
     placement,
     onShow,
+    delayHide,
+    hide,
```

Comment:
I think instead of passing a `hide` value, we should handle that in the caller. Either we have a tooltip, or we don't. So... Better have something like...
```
let barContent = (
    <div className='announcement-bar__text'>
        ...
    </div>
);
if (this.state.showTooltip) {
    barContent = (<WithTooltip ...>{barContent}</WithTooltip>);
}
```

Reply:
Thanks, good idea. I will fix that tomorrow because I'm a little busy today.

Reply:
Done